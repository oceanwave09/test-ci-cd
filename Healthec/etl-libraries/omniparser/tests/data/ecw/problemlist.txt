apuid|RecordID|PatientId|PatientAccountNo|AsmtId|OnsetDate|Code|Name|EncounterId|Status|PtCondition|Risk|AddedDate|ResolvedOn
280807|280807.17502.141518|17502||141518||V70.2|GEN PSYCHIATRIC EXAM NEC|408478|confirmed||||
280807|280807.17502.347783|17502||347783||F80.1|Expressive language disorder|1584628|confirmed|||12/07/2018|
280807|280807.17502.348292|17502||348292||G80.0|Spastic quadriplegic cerebral palsy|1470339|confirmed|||05/06/2018|
280807|280807.17502.356482|17502||356482||M24.511|Contracture, right shoulder|1589618|confirmed|||02/13/2019|
280807|280807.17502.356483|17502||356483||M24.512|Contracture, left shoulder|1589618|confirmed|||02/13/2019|
280807|280807.17502.356488|17502||356488||M24.531|Contracture, right wrist|1589618|confirmed|||02/13/2019|
280807|280807.17502.356489|17502||356489||M24.532|Contracture, left wrist|1589618|confirmed|||02/13/2019|
280807|280807.17502.356491|17502||356491||M24.541|Contracture, right hand|1589618|confirmed|||02/13/2019|
280807|280807.17502.356492|17502||356492||M24.542|Contracture, left hand|1589618|confirmed|||02/13/2019|
280807|280807.17502.414387|17502||414387||R27.8|Coordination abnormal|1589618|confirmed|||02/13/2019|
280807|280807.17502.414875|17502||414875||M24.50|Contracture of joint|1470339|confirmed|||05/06/2018|
280807|280807.17502.414879|17502||414879||M24.522|Contracture of left elbow|1589618|confirmed|||02/13/2019|
280807|280807.17502.414880|17502||414880||M24.521|Contracture of right elbow|1589618|confirmed|||02/13/2019|
280807|280807.17502.415979|17502||415979||F79|Intellectual disability|1470339|confirmed|||05/06/2018|
280807|280807.17502.416220|17502||416220||G80.0|CP (cerebral palsy), spastic, quadriplegic|2268956|confirmed|||10/12/2021|
280807|280807.17502.445589|17502||445589||G80.9|Cerebral Palsy|2289818|confirmed|||12/07/2021|
280807|280807.17502.445594|17502||445594||F79|Unspecified Intellectual Disability|2289818|confirmed|||12/07/2021|
280807|280807.17502.670050|17502||670050||J30.9|Allergic rhinitis, unspecified seasonality, unspecified trigger|2301648|confirmed|||12/14/2021|
280807|280807.17502.677581|17502||677581||Z88.0|Allergy history, penicillin|2301648|confirmed|||12/14/2021|
280807|280807.17502.347012|17502||347012||E78.2|Mixed hyperlipidemia|1357505|confirmed|||11/14/2017|
280807|280807.17502.347653|17502||347653||F41.1|Generalized anxiety disorder|1411793|confirmed|||01/31/2018|
280807|280807.17502.347776|17502||347776||F70|Mild intellectual disabilities|1164954|confirmed|||12/20/2016|
280807|280807.17502.350770|17502||350770||H54.10|Blindness, one eye, low vision other eye, unspecified eyes|2347774|confirmed|||03/24/2022|
280807|280807.17502.360807|17502||360807||Z23|Need for MMR vaccine|1577646|confirmed|||02/20/2019|
280807|280807.17502.361089|17502||361089||N13.39|Other hydronephrosis|2359600|confirmed|||02/07/2022|
280807|280807.17502.414594|17502||414594||R94.31|Abnormal EKG|2280595|confirmed|||01/26/2022|
280807|280807.17502.414722|17502||414722||N91.2|Amenorrhea|1199446|confirmed|||02/07/2017|
280807|280807.17502.415027|17502||415027||F41.0|Panic disorder without agoraphobia|1119307|confirmed|||08/22/2016|
280807|280807.17502.415128|17502||415128||E04.1|Thyroid nodule|1485786|confirmed|||06/13/2018|
