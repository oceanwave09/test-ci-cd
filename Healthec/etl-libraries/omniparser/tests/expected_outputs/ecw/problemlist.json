[
	{
		"apuid": "280807",
		"asmt_id": "141518",
		"code": "V70.2",
		"encounter_id": "408478",
		"name": "GEN PSYCHIATRIC EXAM NEC",
		"patient_id": "17502",
		"record_id": "280807.17502.141518",
		"row_id": "481f4809-4cc7-46eb-98bb-efee31038d99",
		"status": "confirmed"
	},
	{
		"added_date": "12/07/2018",
		"apuid": "280807",
		"asmt_id": "347783",
		"code": "F80.1",
		"encounter_id": "1584628",
		"name": "Expressive language disorder",
		"patient_id": "17502",
		"record_id": "280807.17502.347783",
		"row_id": "4d2c7dd1-3c02-4ce2-a720-60d332266ebe",
		"status": "confirmed"
	},
	{
		"added_date": "05/06/2018",
		"apuid": "280807",
		"asmt_id": "348292",
		"code": "G80.0",
		"encounter_id": "1470339",
		"name": "Spastic quadriplegic cerebral palsy",
		"patient_id": "17502",
		"record_id": "280807.17502.348292",
		"row_id": "6b8ee02d-918e-470c-98f5-0f0b6deb226b",
		"status": "confirmed"
	},
	{
		"added_date": "02/13/2019",
		"apuid": "280807",
		"asmt_id": "356482",
		"code": "M24.511",
		"encounter_id": "1589618",
		"name": "Contracture, right shoulder",
		"patient_id": "17502",
		"record_id": "280807.17502.356482",
		"row_id": "5794f3b6-45c0-4c2f-8e97-7e3b61b95826",
		"status": "confirmed"
	},
	{
		"added_date": "02/13/2019",
		"apuid": "280807",
		"asmt_id": "356483",
		"code": "M24.512",
		"encounter_id": "1589618",
		"name": "Contracture, left shoulder",
		"patient_id": "17502",
		"record_id": "280807.17502.356483",
		"row_id": "0b3e9b30-97e3-4f0b-8a53-4fd4877f7bcc",
		"status": "confirmed"
	},
	{
		"added_date": "02/13/2019",
		"apuid": "280807",
		"asmt_id": "356488",
		"code": "M24.531",
		"encounter_id": "1589618",
		"name": "Contracture, right wrist",
		"patient_id": "17502",
		"record_id": "280807.17502.356488",
		"row_id": "4504e84d-1a3b-4ad3-81cf-203c34b643ff",
		"status": "confirmed"
	},
	{
		"added_date": "02/13/2019",
		"apuid": "280807",
		"asmt_id": "356489",
		"code": "M24.532",
		"encounter_id": "1589618",
		"name": "Contracture, left wrist",
		"patient_id": "17502",
		"record_id": "280807.17502.356489",
		"row_id": "8b4f61e1-d49d-417d-ab0d-070bff8d4747",
		"status": "confirmed"
	},
	{
		"added_date": "02/13/2019",
		"apuid": "280807",
		"asmt_id": "356491",
		"code": "M24.541",
		"encounter_id": "1589618",
		"name": "Contracture, right hand",
		"patient_id": "17502",
		"record_id": "280807.17502.356491",
		"row_id": "e9352970-d63c-4eb8-b039-a01f5a3ae4be",
		"status": "confirmed"
	},
	{
		"added_date": "02/13/2019",
		"apuid": "280807",
		"asmt_id": "356492",
		"code": "M24.542",
		"encounter_id": "1589618",
		"name": "Contracture, left hand",
		"patient_id": "17502",
		"record_id": "280807.17502.356492",
		"row_id": "32797305-05af-4bfd-9cae-e35ca2911807",
		"status": "confirmed"
	},
	{
		"added_date": "02/13/2019",
		"apuid": "280807",
		"asmt_id": "414387",
		"code": "R27.8",
		"encounter_id": "1589618",
		"name": "Coordination abnormal",
		"patient_id": "17502",
		"record_id": "280807.17502.414387",
		"row_id": "48b61c82-94c2-440d-b497-0dcbc165972b",
		"status": "confirmed"
	},
	{
		"added_date": "05/06/2018",
		"apuid": "280807",
		"asmt_id": "414875",
		"code": "M24.50",
		"encounter_id": "1470339",
		"name": "Contracture of joint",
		"patient_id": "17502",
		"record_id": "280807.17502.414875",
		"row_id": "b20ca05c-4ba1-4a09-b4de-b1002c7a6957",
		"status": "confirmed"
	},
	{
		"added_date": "02/13/2019",
		"apuid": "280807",
		"asmt_id": "414879",
		"code": "M24.522",
		"encounter_id": "1589618",
		"name": "Contracture of left elbow",
		"patient_id": "17502",
		"record_id": "280807.17502.414879",
		"row_id": "12ebb7a1-6524-4918-ae75-d345f2d92cbe",
		"status": "confirmed"
	},
	{
		"added_date": "02/13/2019",
		"apuid": "280807",
		"asmt_id": "414880",
		"code": "M24.521",
		"encounter_id": "1589618",
		"name": "Contracture of right elbow",
		"patient_id": "17502",
		"record_id": "280807.17502.414880",
		"row_id": "26b335cb-d63c-412f-b69a-293e56d2edf7",
		"status": "confirmed"
	},
	{
		"added_date": "05/06/2018",
		"apuid": "280807",
		"asmt_id": "415979",
		"code": "F79",
		"encounter_id": "1470339",
		"name": "Intellectual disability",
		"patient_id": "17502",
		"record_id": "280807.17502.415979",
		"row_id": "7aca1ef4-d641-446f-9f41-5cb9602a0a8c",
		"status": "confirmed"
	},
	{
		"added_date": "10/12/2021",
		"apuid": "280807",
		"asmt_id": "416220",
		"code": "G80.0",
		"encounter_id": "2268956",
		"name": "CP (cerebral palsy), spastic, quadriplegic",
		"patient_id": "17502",
		"record_id": "280807.17502.416220",
		"row_id": "e4504d21-8abd-4826-bf74-1f2f8cf4bf8a",
		"status": "confirmed"
	},
	{
		"added_date": "12/07/2021",
		"apuid": "280807",
		"asmt_id": "445589",
		"code": "G80.9",
		"encounter_id": "2289818",
		"name": "Cerebral Palsy",
		"patient_id": "17502",
		"record_id": "280807.17502.445589",
		"row_id": "a24fe511-0708-49b9-a084-48fc942eef4c",
		"status": "confirmed"
	},
	{
		"added_date": "12/07/2021",
		"apuid": "280807",
		"asmt_id": "445594",
		"code": "F79",
		"encounter_id": "2289818",
		"name": "Unspecified Intellectual Disability",
		"patient_id": "17502",
		"record_id": "280807.17502.445594",
		"row_id": "eab1f07a-c85f-48ad-ac14-7f026548f612",
		"status": "confirmed"
	},
	{
		"added_date": "12/14/2021",
		"apuid": "280807",
		"asmt_id": "670050",
		"code": "J30.9",
		"encounter_id": "2301648",
		"name": "Allergic rhinitis, unspecified seasonality, unspecified trigger",
		"patient_id": "17502",
		"record_id": "280807.17502.670050",
		"row_id": "4216eafc-3f38-4d9e-906e-cc386ab6dc77",
		"status": "confirmed"
	},
	{
		"added_date": "12/14/2021",
		"apuid": "280807",
		"asmt_id": "677581",
		"code": "Z88.0",
		"encounter_id": "2301648",
		"name": "Allergy history, penicillin",
		"patient_id": "17502",
		"record_id": "280807.17502.677581",
		"row_id": "838c4640-7990-4018-ad87-7e8ca3e2e1da",
		"status": "confirmed"
	},
	{
		"added_date": "11/14/2017",
		"apuid": "280807",
		"asmt_id": "347012",
		"code": "E78.2",
		"encounter_id": "1357505",
		"name": "Mixed hyperlipidemia",
		"patient_id": "17502",
		"record_id": "280807.17502.347012",
		"row_id": "ace71649-d3c1-4023-a6d3-d52e17edd172",
		"status": "confirmed"
	},
	{
		"added_date": "01/31/2018",
		"apuid": "280807",
		"asmt_id": "347653",
		"code": "F41.1",
		"encounter_id": "1411793",
		"name": "Generalized anxiety disorder",
		"patient_id": "17502",
		"record_id": "280807.17502.347653",
		"row_id": "26c5b824-2397-45f8-a26a-aa6563fca831",
		"status": "confirmed"
	},
	{
		"added_date": "12/20/2016",
		"apuid": "280807",
		"asmt_id": "347776",
		"code": "F70",
		"encounter_id": "1164954",
		"name": "Mild intellectual disabilities",
		"patient_id": "17502",
		"record_id": "280807.17502.347776",
		"row_id": "a17664c4-b768-424c-8547-c08cc5091c65",
		"status": "confirmed"
	},
	{
		"added_date": "03/24/2022",
		"apuid": "280807",
		"asmt_id": "350770",
		"code": "H54.10",
		"encounter_id": "2347774",
		"name": "Blindness, one eye, low vision other eye, unspecified eyes",
		"patient_id": "17502",
		"record_id": "280807.17502.350770",
		"row_id": "8e0aca73-e122-4e76-945c-0406d9c12344",
		"status": "confirmed"
	},
	{
		"added_date": "02/20/2019",
		"apuid": "280807",
		"asmt_id": "360807",
		"code": "Z23",
		"encounter_id": "1577646",
		"name": "Need for MMR vaccine",
		"patient_id": "17502",
		"record_id": "280807.17502.360807",
		"row_id": "f76a025d-48f4-44c9-83fb-dfbdf6433350",
		"status": "confirmed"
	},
	{
		"added_date": "02/07/2022",
		"apuid": "280807",
		"asmt_id": "361089",
		"code": "N13.39",
		"encounter_id": "2359600",
		"name": "Other hydronephrosis",
		"patient_id": "17502",
		"record_id": "280807.17502.361089",
		"row_id": "74418c9c-edd0-422a-8731-03f53a494a7e",
		"status": "confirmed"
	},
	{
		"added_date": "01/26/2022",
		"apuid": "280807",
		"asmt_id": "414594",
		"code": "R94.31",
		"encounter_id": "2280595",
		"name": "Abnormal EKG",
		"patient_id": "17502",
		"record_id": "280807.17502.414594",
		"row_id": "af35636d-7a6e-4289-b7be-c992acb97a88",
		"status": "confirmed"
	},
	{
		"added_date": "02/07/2017",
		"apuid": "280807",
		"asmt_id": "414722",
		"code": "N91.2",
		"encounter_id": "1199446",
		"name": "Amenorrhea",
		"patient_id": "17502",
		"record_id": "280807.17502.414722",
		"row_id": "f20f9537-610c-4273-ab01-2ae60dee5d29",
		"status": "confirmed"
	},
	{
		"added_date": "08/22/2016",
		"apuid": "280807",
		"asmt_id": "415027",
		"code": "F41.0",
		"encounter_id": "1119307",
		"name": "Panic disorder without agoraphobia",
		"patient_id": "17502",
		"record_id": "280807.17502.415027",
		"row_id": "019564d4-5a1d-4d00-807f-2a4abd81a9a1",
		"status": "confirmed"
	},
	{
		"added_date": "06/13/2018",
		"apuid": "280807",
		"asmt_id": "415128",
		"code": "E04.1",
		"encounter_id": "1485786",
		"name": "Thyroid nodule",
		"patient_id": "17502",
		"record_id": "280807.17502.415128",
		"row_id": "670f26f1-e910-44d3-87fa-34696c93f1cc",
		"status": "confirmed"
	}
]
